scene/2d/joints_2d.cpp

```diff
@@ -32,8 +32,33 @@
 
 #include "core/config/engine.h"
 #include "physics_body_2d.h"
+#include "scene/scene_string_names.h"
 #include "servers/physics_server_2d.h"
 
+void Joint2D::_disconnect_signals() {
```

Comment:
`_disconnect_signals`also needs to be called in `NOTIFICATION_EXIT_TREE` for the joint (in 3D as well), to avoid cases where it's called later when the joint is not in the tree anymore, which causes errors with `get_node` (case where the joint is removed, then the bodies).

Reply:
Done.